tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java

```diff
@@ -97,21 +97,22 @@ public void run() {
   private static final int TEST_NUM_MASTERS = 3;
   private static final long TEST_TIME_MS = Constants.SECOND_MS;
 
-  private ClientThread mCreateFileThread = null;
+  private ClientThread mCreateFileThread;
   /** Executor for running client threads. */
-  private final ExecutorService mExecutorsForClient = Executors.newFixedThreadPool(1);
-  private Configuration mMasterConfiguration = null;
+  private ExecutorService mExecutorsForClient;
+
+  public SystemPropertyRule mDisableHdfsCacheRule =
```

Comment:
is this the way we are supposed to use `SystemPropertyRule`?
shall we declare it as `@ClassRule`?